### PR TITLE
Fixed slides and made the build work

### PR DESCRIPTION
**Pull request by ElijahCaine**
The build was throwing an error when I ran scripts/localbuild.sh so I fixed that and modfiied the formatting of a few slides.

### CODE REVIEW

File: source/daycamp/daycamp_session1.rst [hunk 1]
```diff
@@ -238,6 +238,7 @@ A local or remote instance of an operating system typically used for shared
 resoruces and/or shared applications.
```
Comment (dspt) [inline source/daycamp/daycamp_session1.rst]: typo

**Comment by ElijahCaine** [comment]
@dspt @lucywyman thoughts?

**Comment by dspt** [comment]
:+1:

**Comment by lucywyman** [comment]
+1

**Pull request merged**